File: Leetcode/36.cpp
class Solution {
public:
    void sortColors(vector<int>& nums) {
    int i, cnt0=0, cnt1=0, cnt2=0;
    int n=nums.size();
    for(i=0; i<n; i++){
        if(nums[i]==0){
            cnt0++;
        }else if(nums[i]==1){
            cnt1++;
        }else if(nums[i]==2){
            cnt2++;
        }
    }
    i=0;
    while (cnt0!=0 || cnt1!=0 || cnt2!=0){
        if (cnt0!=0){
            nums[i++]=0;
            cnt0--;
        }else if(cnt1!=0){
            nums[i++]=1;
            cnt1--;
        }else if(cnt2!=0){
            nums[i++]=2;
            cnt2--;
            }
        }
    }
};

File: Leetcode/36.py
class Solution:
    def sortColors(self, nums: List[int]) -> None:
        """
        Do not return anything, modify nums in-place instead.
        """
        cnt0=0
        cnt1=0
        cnt2=0
        for i in range(len(nums)):
            if nums[i]==0:
                cnt0+=1
            elif nums[i]==1:
                cnt1+=1
            elif nums[i]==2:
                cnt2+=1
        for i in range(len(nums)):
            if (cnt0>0):
                nums[i]=0
                cnt0-=1
            elif (cnt1>0):
                nums[i]=1
                cnt1-=1
            elif (cnt2>0):
                nums[i]=2
                cnt2-=1
            